saleor/plugins/webhook/tests/subscription_webhooks/test_create_deliveries_for_subscription.py

```diff
@@ -1546,6 +1547,227 @@ def test_voucher_deleted(voucher, subscription_voucher_deleted_webhook):
     assert deliveries[0].webhook == webhooks[0]
 
 
+def test_payment_authorize(payment, subscription_payment_authorize_webhook):
```

Comment:
IMHO this file is growing very fast. 
I would propose to put all tests related to payments into single file.
And for the rest of the sync webhooks - one file per one event, this should help us in keeping readability.

Reply:
I totally agree - though I would leave it for another PR to make review easier and shiorter ATM